packages/react-examples/src/react-experiments/Toggle/TogglePage.tsx

```diff
@@ -48,14 +48,14 @@ export class TogglePage extends React.Component<IComponentDemoPageProps, {}> {
           <Markdown>
```

Comment:
Wrong file (sorry) but you should look at whether there are any files under `react-examples/src/react-{whatever}` that should be moved or removed:
- example files
- doc markdown files
- next example files
- .doc.tsx (this is needed for current but not for next)

and all corresponding path updates

(edit: after writing this I see that some of the files were renamed and show up later, so maybe it's fine)

Reply:
Yes, I went through all of them, I think some of them had the same naming scheme for `next` and `default` examples so they only show as changes even though I moved them? I wish GitHub kept track of what these moves were step by step instead of batching them all up into the "final" diffs it shows.